code/game/objects/structures/displaycase.dm

```diff
@@ -336,6 +350,10 @@
 		else
 			..()
 
+/obj/item/key/displaycase

```

Comment:
The `key` typepath is really weird, because right now the keys are only used in vehicles, and this key will probably be able to be used on an atv.